WcaOnRails/config/locales/en.yml

```diff
@@ -49,6 +49,7 @@ en:
         use_wca_registration: "I would like to use the WCA website for registration"
         guests_enabled: "Guests"
         receive_registration_emails: "Notify me by email when someone registers for this competition"
+        friendly_id: "Public facing ID"
```

Comment:
I'm not sure either. The common technical term is ["slug"](https://en.wikipedia.org/wiki/Semantic_URL#Slug), but there are probably not many people who know what it means.